test/AWS.Deploy.CLI.IntegrationTests/WebAppNoDockerFileTests.cs

```diff
@@ -80,7 +80,7 @@ public async Task DefaultConfigurations()
             var deployStdOut = _interactiveService.StdOutReader.ReadAllLines();
 
             // Example:     Endpoint: http://52.36.216.238/
-            var applicationUrl = deployStdOut.First(line => line.StartsWith($"\tEndpoint"))
+            var applicationUrl = deployStdOut.First(line => line.Trim().StartsWith($"Endpoint"))
```

Comment:
I believe this is done in multiple tests. Can you make sure the rest of the tests do it the same way?

Reply:
I can do that later when all of the PRs are merged together. I'm not sure I believe this is a useful utility method that has single line.